### ISSUE TITLE
feature for dumb people

### ISSUE THREAD

**Issue by OrionRandD** [jitsi/jitsi-meet#10931]
I am trying to teach jitsi-meet for some people, but there are some very difficult to learn.
When they open a meeting and do not set a password to protect the meeting, jitsi warns that the meeting is unsecure. 
But it does not say to the person who openned th meeting that he/she needs to set a password to protect the meeting...
Can you to improve this warning? Say: pointing to where he/she should set the password? 
As a teacher, I have observed that people are afraid of clicking in any program settings and messing them. Non-technical people should be taken into more consideration when trying to use jitsi...
Also, a good idea would be not to allow any meeting to be started without setting a password first... For that, the user should be lead straight away to set a meeting password after opening any jitsi session...

**Comment by damencho** [comment]
Please, when you have questions or problems use the community forum before opening new issues, thank you.

The insecure meeting warning is that the room name you use is not secure and is from common words or can be easily guessed. If you use the random room name generator on the welcome page to create links, you can avoid this warning and then use more secure meeting names. 
More on the subject https://jitsi.org/security/

On a platform like meet.jit.si we cannot pre-set password as there is no room reservation and no authentication to allow that. In cases like 8x8.vc and jaas you can have that. You can subscribe for 8x8 Meet Pro and have your personal room and pre-set a password for it, or pre-enable lobby for it.

**Comment by OrionRandD** [comment]
Thx for the clarification...
I thought that the warning was only related to not setting a password,
but it has to do with weakly naming a room...
By the way, instead of a forum, do you know any official IRC channel for
jitsi-meeting?

----------------------------------

**Comment by damencho** [comment]
The forum is our preferred channel of communication.

**Comment by complabs28** [comment]
You should also enable the closed captions feature in jitsi meet for those who cannot hear

**Comment by kkarhan** [comment]
@saghul @damencho I have a question: Is *closed caption* implemented? 
Cuz I see this issue marked with *closed as completed* and for many people this is [a *non-negotiable* accessibility feature](
https://dair-community.social/@tomstoneham/112132440737732711) ...

**Comment by saghul** [comment]
Yes they are. You need to either use JaaS or your own deployment. We don't offer them for free on meet.jit.si. The backend, jigasi, is open source.

**Comment by kkarhan** [comment]
@saghul interesting...
Is that documented *anywhere*?

Please feel free to post a link so I can forward it to the [original asking user](https://dair-community.social/@tomstoneham/112132440737732711)...

**Comment by damencho** [comment]
Yes in the official install guide:

https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart/#install-jigasi
